README.md

```diff
@@ -22,13 +22,24 @@ Supported Cars
 - Acura ILX 2016 with AcuraWatch Plus
   - Due to use of the cruise control for gas, it can only be enabled above 25 mph
 
+- Honda Accord 2018 with Honda Sensing (alpha!)
+  - Uses stock Honda Sensing for longitudinal control
```

Comment:
This is true as well, right? @energee 
- Due to limitations in steering firmware, steering is disabled below 12 mph

Reply:
I don't think so, I believe accord steers down to 0.

Reply:
The Accord steers down to around 1.2 m/s. Below that number `STEER_STATUS` changes to 3 

![image](https://user-images.githubusercontent.com/1934727/41048713-34c2ae3e-6964-11e8-83ba-2c50c1344714.png)